server/amp/helpers/get-amp-page-base-path.js

```diff
@@ -0,0 +1,10 @@
+const get = require("lodash/get");
+
+function getAmpPageBasePath(opts = {}) {
+  let ampPageBasePath = get(opts, ["featureConfig", "ampPageBasePath"], "/amp/story");
+  ampPageBasePath = typeof ampPageBasePath === "function" ? ampPageBasePath() : ampPageBasePath;
```

Comment:
You'll need to pass `config` to `ampPageBasePath` function.. Madrid will need config to access pagebuilder config

Madrid code:
![Screenshot 2023-11-29 at 11 33 21 AM](https://github.com/quintype/quintype-node-framework/assets/27222520/450af40c-eb25-4706-880e-277100ffbe02)